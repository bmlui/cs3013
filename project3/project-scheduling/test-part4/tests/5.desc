analysis: RR: support different time slice
