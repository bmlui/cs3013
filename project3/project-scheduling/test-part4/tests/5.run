./scheduler 1 RR tests/5.in 4
