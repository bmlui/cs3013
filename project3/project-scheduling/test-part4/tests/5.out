Execution trace with RR:
t=0: [Job 0] arrived at [0], ran for: [1]
t=3: [Job 1] arrived at [3], ran for: [4]
t=7: [Job 1] arrived at [3], ran for: [1]
End of execution with RR.
Begin analyzing RR:
Job 0 -- Response time: 0  Turnaround: 1  Wait: 0
Job 1 -- Response time: 0  Turnaround: 5  Wait: 0
Average -- Response: 0.00  Turnaround 3.00  Wait 0.00
End analyzing RR.
